content/_help/specific-agencies/ssa._en.md

```diff
@@ -8,20 +8,20 @@ order: 3
 redirect_from:
   - /en/help/specific-agencies/ssa/
 ---
-Login.gov is for secure sign in only. Your Login.gov account does not affect or have any information about your [Social Security](https://www.ssa.gov/) card, number, or benefits. 
+Login.gov is for secure sign in only. Your Login.gov account does not affect or have any information about your Social Security card, number, or benefits. 
 
 Please do not send Login.gov sensitive data about yourself or identifying numbers.
 
 Login.gov can only answer questions about the sign-in process and creating a Login.gov account.
 
-If you already have an [SSA.gov](https://www.ssa.gov/) account, you do not need to create a Login.gov account unless required by SSA.
+If you created a Social Security username more than 3 years ago, you will need to transition to a new or existing Login.gov account to have continuous access to SSA online services. For step-by-step instructions on how to make the transition, watch this [video](https://www.youtube.com/watch?v=AMADyonYNTE).
```

Comment:
I know this has already gone through translation, but IMO I think the link might be better if it included an action in the text too for context
```suggestion
If you created a Social Security username more than 3 years ago, you will need to transition to a new or existing Login.gov account to have continuous access to SSA online services. For step-by-step instructions on how to make the transition, [watch this video](https://www.youtube.com/watch?v=AMADyonYNTE).
```

Reply:
I'll flag this for further review but merge the changes as they are for now.